app/src/test/java/bitmap/transformer/BitMapTest.java

```diff
@@ -1,4 +1,25 @@
 package bitmap.transformer;
 
+import org.junit.Test;
+import java.awt.*;
+import java.util.ArrayList;
+import static org.junit.Assert.assertEquals;
+
 public class BitMapTest {
```

Comment:
Good work on your tests. If you are ever unsure, you can always test happy path and empty path!